portal/src/main/webapp/WEB-INF/jsp/patient_view/patient_view.jsp

```diff
@@ -125,7 +125,7 @@ boolean showDrugs = GlobalProperties.showDrugsTab();
 boolean showSamplesTable = isPatientView;
 String userName = GlobalProperties.getAuthenticatedUserName();
 
-String darwinAccessURL = CheckDarwinAccess.checkAccess(request);
+// String darwinAccessURL = CheckDarwinAccess.checkAccess(request);
```

Comment:
Is this comment necessary? Is it an old temporary fix for the issue already solved by PR #1389?